app/src/pages/inside/common/externalLink/externalLink.scss

```
@@ -49,14 +50,25 @@
-    padding-bottom: 3px;
 
     svg {
       width: 13px;
       height: 13px;
     }
+
+    &.default-color {
+      color: var(--rp-ui-color-field-icon);
+
+      svg path {
+        fill: currentColor;
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)